### PR TITLE
Added my sub-folder and two files to help people shim their MGN7H bearing cartridges.

### PR DESCRIPTION
**Pull request by lococnc**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [ ] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ like so:
 
 | Creator | Mod title | Description | V0 | V1.8 | V2.4 | VSW |
 | --- | --- | --- | --- | --- | --- | --- |
+|LoCoCNC| [MGN7H Shims](./LoCoCNC/) | Shims for making bearing cartridges coplanar |:heavy_check_mark:|:x:|:x:|:x:|
```
Comment (KiloQubit) [inline printer_mods/README.md:28]: Please move this down the list so it's in alphabetical order, looks like your line should be after `littleferris`.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

- Please create another subdirectory for your mod inside `LoCoCNC`, in case you ever have other mods.  So move this mod into `printer_mods/LoCoCNC/MGN7H_Shims/` for example
- Your STL files are not in a printable orientation, please rotate them.
- Fix the oder of mods in the README as noted  below.

**Comment by lococnc** [comment]
Updated as requested.

File: printer_mods/README.md [hunk 2]
```diff
@@ -115,6 +114,7 @@ like so:
 | lakai | [Bruag Skirts](./lakai/Bruag_Skirts) | V2 350mm skirts inspired by Bruag Facades | :x: | :x: | :heavy_check_mark: | :x: |
 |leandromarceddu|[PowerSkirt](./leandromarceddu/PowerSkirt) |Modified skirtpiece for a digital power monitor|:x:|:grey_question:|:heavy_check_mark:|:grey_question:|
 |littleferris |[V0 c14 Skirt Inlet](./littleferris/V0_Skirt_Inlet) |V0 C14 Skirt inlet |:heavy_check_mark: |:x: |:x: |:x:|
+|LoCoCNC| [MGN7H Shims](./LoCoCNC/) | Shims for making bearing cartridges coplanar |:heavy_check_mark:|:x:|:x:|:x:|
```
Comment (KiloQubit) [inline printer_mods/README.md:117]: Update this link to point to your mod in the new directory

    [MGN7H Shims](./LoCoCNC/MGN7H_Shims)

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the updates!  Please see one more change below.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**